src/services/navigationBar.ts

```diff
@@ -700,4 +703,16 @@ namespace ts.NavigationBar {
                 return false;
         }
     }
+
+    function cleanText(text: string): string {

```

Comment:
I think you were planning to add a comment to help consumers decide whether or not to call this helper.

Reply:
This method should be called for every string that is returned. I think the comment is unnecessary, as the method is to small to describe itself. Let me know if you think otherwise.